faf-java-server-app/src/main/java/com/faforever/server/security/Login.java

```diff
@@ -52,6 +55,10 @@
   @Column(name = "ip")
   private String ip;
 
+  @Temporal(TemporalType.TIMESTAMP)
+  @Column(name = "last_login")
+  private Date lastLogin;
```

Comment:
`java.util.Date` is not used in this project as it was superceeded by the `java.time` package. Please use `java.time.OffsetDateTime` (the `@Temporal` is not required on java.time types)
See `AbstractEntity` as a reference.